front/src/modules/simulationResult/components/SpaceCurvesSlopes.tsx

```diff
@@ -238,8 +250,7 @@ const SpaceCurvesSlopes = ({
     enableInteractivity(
       chartLocal,
       trainData,
-      ['position', 'gradient'],
-      ['slopesCurve'],
+      keyValues,
```

Comment:
Use directly the constant, it's more readable
```suggestion
      CHART_AXES.SPACE_GRADIENT,
```

Reply:
done